synAnnotationUtils/update.py

```diff
@@ -4,8 +4,12 @@
 import logging
 
 
-# Update Annotations 
-## by dict
+'''
+	 Updates File entity Annotations
```

Comment:
make this [PEP 257](https://www.python.org/dev/peps/pep-0257/) compliant, please!